bin/telemetry_sim_ATHS

```diff
@@ -66,11 +66,15 @@ def cmdline():
                         help="The tracking coordinate system [Local, Sidereal, or non-sidereal].")
     parser.add_argument("--radesys", action="store", default="ICRS", type=str,
                         help="The ra/dec track system value.")
-    parser.add_argument("--object", action="store", default="ElGrodo", type=str,
+    parser.add_argument("--object", action="store", default="ElGordo", type=str,
```

Comment:
Can you please use a meaningful default here and for imagetag below?

Reply:
"El Gordo" is largest galaxy cluster at high redshift, see: https://en.wikipedia.org/wiki/El_Gordo_(galaxy_cluster)

Reply:
that may well be but "DEFAULTOBJECT" and "DEFAULTIMAGETAG" would be more obvious (and ImageTag should never be a galaxy).